src/Plugin.Maui.ScreenRecording/ScreenRecordingOptions.cs

```diff
@@ -21,4 +21,18 @@ public class ScreenRecordingOptions
 	/// Default value is <see langword="false"/>.
 	/// </summary>
 	public bool EnableMicrophone { get; set; }
+
+	/// <summary>
+	/// Gets or sets the notification content title.
+	/// Default value is "Screen Recording".
+	/// </summary>
+	/// <remarks>Currently only implemented for Android.</remarks>
```

Comment:
```suggestion
	/// <remarks>This property only has effect on Android. On other platforms no notification is shown when a screen recording is being made.</remarks>
```

Changed the message a little. I doubt we will implement this for other platforms.

Reply:
Changed